### PR TITLE
xCAT-genesis-builder for Openpower

### PR DESCRIPTION
**Pull request by fenggaobj**
For ticket https://github.com/xcat2/xcat-core/issues/281
Update xCAT-genesis-builder for building MCP 8.1, make it suitable for openpower

### CODE REVIEW

File: xCAT-genesis-builder/buildrpm [hunk 1]
```diff
@@ -10,6 +10,11 @@ DIR=`dirname $0`
 DIR=`readlink -f $DIR`
 BUILDARCH=`uname -m`
 
+#For Openpower
+if [ $BUILDARCH = "ppc64le" ]; then
```
Comment (zet809) [inline xCAT-genesis-builder/buildrpm]: Are you plan to build genesis on ppc64le arch?

Comment (fenggaobj) [inline xCAT-genesis-builder/buildrpm]: We will build genesis using MCP 8.1 on OpenPower, the command "uname -m" returns "ppc64le" on OpenPower.

Comment (fenggaobj) [inline xCAT-genesis-builder/buildrpm]: Only for PCM

**Comment by zet809** [comment]
Seems ok to me. I agree to merge.

**Pull request merged**